courses/free-courses-en.md

```diff
@@ -128,10 +128,12 @@
 * [Codecademy](https://www.codecademy.com)
 * [Coursera](https://www.coursera.org)
 * [Datacamp](https://www.datacamp.com)
+* [Deakin University Free Online Courses](https://www.deakin.edu.au/study-online/free-online-courses)
 * [DevDocs](https://devdocs.io)
 * [edX](https://www.edx.org)
 * [freeCodeCamp](https://www.freecodecamp.org)
 * [FutureLearn](https://www.futurelearn.com)
+* [General Assembly](https://generalassemb.ly)
```

Comment:
not free

Reply:
Some of these were definitely free. Looks like they have removed their free courses. I will fix.